control/autoware_autonomous_emergency_braking/src/node.cpp

```diff
@@ -842,17 +849,13 @@ void AEB::createObjectDataUsingPredictedObjects(
 
         // If the object is behind the ego, we need to use the backward long offset. The
         // distance should be a positive number in any case
-        const bool is_object_in_front_of_ego = obj_arc_length > 0.0;
-        const double dist_ego_to_object =
-          (is_object_in_front_of_ego) ? obj_arc_length - vehicle_info_.max_longitudinal_offset_m
-                                      : obj_arc_length + vehicle_info_.min_longitudinal_offset_m;
+        const double dist_ego_to_object = obj_arc_length - longitudinal_offset;
 
         ObjectData obj;
         obj.stamp = stamp;
         obj.position = obj_position;
         obj.velocity = obj_tangent_velocity;
         obj.distance_to_object = std::abs(dist_ego_to_object);
-        obj.is_target = true;
```

Comment:
Please dont delete this line. It is not require, but makes the code more explicit.

Reply:
@danielsanchezaran 
> Please dont delete this line. It is not require, but makes the code more explicit.

I don't intend to delete this line, This line is deleted in your refactoring branch. 
But ok :+1: and I agree.

Reply:
added in https://github.com/autowarefoundation/autoware.universe/pull/9462/commits/8846adaea8fb23731cade10207c7506636509c72